docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -127,17 +150,72 @@ PAF-of-1 Cause/Risk Pairs
 Risk-Outcome Relationships
 ++++++++++++++++++++++++++
 
-Coverage Gap Framework
-++++++++++++++++++++++
+Stratifying Exposure Variables by Baseline Intervention Coverage
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+From GBD we obtain overall population estimates for our outcomes of interest
+(prevalence of vitamin A deficiency, birth prevalence of neural tube defects,
+distribution of haemoglobin levels, and distribution of birth weights). Because
+there exists baseline coverage of our interventions --- i.e. there is already
+some food fortification occurring in the locations we are simulating --- these
+overall population estimates include both people who are eating fortified food
+and those who are not.
+
+When we introduce additional food fortification into the population, only those
+who are *not* already eating fortified food should receive the benefit.
+Therefore, in order to properly measure the effects of our interventions, we
+must first stratify our population into those who are already covered vs. not
+covered by the forticant of interest. We then need to calculate the risk
```

Comment:
```suggestion
covered by the forticant of interest. We then need to calculate the prevalence of the risk
```

Reply:
since earlier it was referred to as prevalence

Reply:
I was avoiding saying "prevalence" except for specific examples, because the general strategy applies to continuous variables as well as discrete ones. For categorical variables the exposure distribution is specified by the prevalence in each category, but for continuous variables we need the probability density function, which is like a "prevalence density" at each exposure level. I could explicitly say "exposure *distribution*" here though.